man/systemd-networkd-wait-online.service.xml

```diff
@@ -85,6 +85,17 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--any</option></term>
+
+        <listitem><para>Even if several interfaces are in configuring state,
+        <command>systemd-networkd-wait-online</command> exits with success when at least one interface
+        becomes online. When this option is specified with <option>--interface</option>, then
```

Comment:
following our usual syntax please suffix `--interface` with `=` to indicate it expects an argument. i.e. `<option>--interface</option>` → `<option>--interface=</option>`